src/Fetcher/TululaFetcher.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Fetcher;
+
+use App\Entity\Conference;
+use App\Entity\Continent;
+use App\Entity\Tag;
+use App\Repository\ExcludedTagRepository;
+use App\Repository\TagRepository;
+use Psr\Log\LoggerInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class TululaFetcher implements FetcherInterface
+{
+    private const SOURCE = 'tulula';
+    private const TAG_ALLOW_LIST = [
+        'android',
+        'css',
+        'data',
+        'devops',
+        'dotnet',
+        'facebook',
+        'golang',
+        'html',
+        'ios',
+        'java',
+        'javascript',
+        'nodejs',
+        'php',
+        'python',
+        'react native',
+        'ruby',
+        'rust',
+        'security',
+        'ux',
+        'ai/ml',
+        'sql',
+        'iot',
+    ];
+
+    private LoggerInterface $logger;
+    private TagRepository $tagRepository;
+    private ExcludedTagRepository $excludedTagsRepository;
+    private LocationGuesser $locationGuesser;
+    private HttpClientInterface $client;
+
+    public function __construct(LoggerInterface $logger, LocationGuesser $locationGuesser, HttpClientInterface $client, TagRepository $tagRepository, ExcludedTagRepository $excludedTagRepository)
+    {
+        $this->client = $client;
+        $this->logger = $logger;
+        $this->tagRepository = $tagRepository;
+        $this->locationGuesser = $locationGuesser;
+        $this->excludedTagsRepository = $excludedTagRepository;
+    }
```

Comment:
Can you sort every property in the same order 
1. class property
2. constructor argument
3. constructor assignation
I understand this comment could be boring, but it's really easier to edit arguments later on.

And BTW, the Logger should be the last one (almost always) since it should be optional (almost always)
And to avoid checking if a logger exist before using it, we usually use the Null pattern:
```php
public function __construct(LoggerInterface $logger = null)
{
    $this->logger = $logger ?? new NullLogger()?
}
```